assets/css/admin/altadmintheme/login/loginscreens.less

```diff
@@ -9,13 +9,38 @@ html.presidecms {
 	background-size       : var( --login-bg-size ) !important;
 
 	.login-layout {
+
+		.vimeo-wrapper {
+		   position: fixed;
+		   top: 0;
+		   left: 0;
+		   width: 100%;
+		   height: 100%;
+		   z-index: -1;
+		   pointer-events: none;
+		   overflow: hidden;
+
+		   iframe {
+			   width: 100vw;
+			   height: 56.25vw;
+			   min-height: 100vh;
+			   min-width: 177.77vh;
```

Comment:
This is all locked to the video being 16:9 ratio.

Woiuld recommend trying the aspect-ratio CSS property: https://caniuse.com/mdn-css_properties_aspect-ratio

So, lose the width and height, have min-width and min-height as 100vw and 100vh, and then set the aspect ratio directly in the HTML of the iframe, so it could be pulled from a setting alongside the video URL.